go/libkb/proof_support_coinbase.go

```diff
@@ -83,6 +84,29 @@ func (t CoinbaseServiceType) CheckUsername(s string) (err error) {
 	return
 }
 
+func coinbaseUserURL(s string) string {
+	return "https://coinbase.com/" + s
+}
+
+func coinbaseSettingsURL(s string) string {
```

Comment:
lowercase = not exported outside of the library

uppercase = exported outside of the library

weird, right?

On Sat, Dec 12, 2015 at 10:44 AM, Chris Coyne <notifications@github.com>
wrote:

> In go/libkb/proof_support_coinbase.go
> <https://github.com/keybase/client/pull/1521#discussion_r47433066>:
>
> > @@ -83,6 +84,29 @@ func (t CoinbaseServiceType) CheckUsername(s string) (err error) {
> >  	return
> >  }
> >
> > +func coinbaseUserURL(s string) string {
> > +	return "https://coinbase.com/" + s
> > +}
> > +
> > +func coinbaseSettingsURL(s string) string {
>
> looking at other functions here, should it be "CoinbaseSettingsURL()" not
> "coinbaseSettingsURL()" ? Just curious....I don't know what the preferred
> Go style is...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1521/files#r47433066>.
>